code/_globalvars/lists/flavor_misc.dm

```diff
@@ -160,7 +160,8 @@ GLOBAL_LIST_INIT(jumpsuitlist, list(PREF_SUIT, PREF_SKIRT))
 #define UPLINK_PDA		"PDA"
 #define UPLINK_RADIO	"Radio"
 #define UPLINK_PEN		"Pen" //like a real spy!
-GLOBAL_LIST_INIT(uplink_spawn_loc_list, list(UPLINK_PDA, UPLINK_RADIO, UPLINK_PEN))
+#define UPLINK_IMPLANT  "Implant"  // expensive, but unstealable

```

Comment:
Please document this as you would document a variable with dmdoc to be compliant with our requirements for documentation. [More information can be found here.](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md)